og/bump/action.yml

```diff
@@ -12,22 +12,27 @@ inputs:
   pr_title:
     description: SC formatted title of the PR.
     required: true
+  pr_number:
+    description: PR serial number
+    required: true
+  github_repository:
+    descripton: Github repository 
+    required: true
   dev_branch:
     description: The dev branch of this action to connect to the workflow
     required: false
     default: 'main'
 
 runs:
   using: "composite"
-
   steps:
     - run: python3 -m pip install semver
       shell: bash
-    
+      
     - run: |
-        cat << \EOF | tee /tmp/base_branch
-          ${{ inputs.base_branch }}
-        EOF
+        export BASE_BRANCH=${{ inputs.base_branch }}
+        export PR_NUMBER=${{ inputs.pr_number }}
+        export GITHUB_REPOSITORY=${{ inputs.github_repository }}
         cat << \EOF | tee /tmp/pr_body
```

Comment:
I guess it's only worth exporting those variables that will be used in the CHANGELOG?

Reply:
This workflow is also used for bumping the version. Exported are environment variables that do not include special characters that need escaping, for those we use escaped `heredoc` (`\EOF`). This might be a good opportunity to change the delimiter as well, from `EOF` to something less likely to end up in a PR description 🤔